### PR TITLE
ly scribble

### PR DESCRIPTION
**Pull request by Ly900**
Completeness: 3
Comfort: 3

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class CommentsController < ApplicationController
```
Comment (andrewsunglaekim) [inline app/controllers/comments_controller.rb:1]: encourage you to finish out comment functionality when you get a chance!

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+class PostsController < ApplicationController
```
Comment (andrewsunglaekim) [inline app/controllers/posts_controller.rb:1]: :+1:

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<h2>
```
Comment (andrewsunglaekim) [inline app/views/posts/show.html.erb:1]: :+1:

File: db/migrate/20151109231623_create_blogs.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class CreateBlogs < ActiveRecord::Migration
```
Comment (andrewsunglaekim) [inline db/migrate/20151109231623_create_blogs.rb:1]: migrations look great :+1:

**Comment by andrewsunglaekim** [comment]
Views and controller actions for posts look great. I encourage you to implement the commenting functionality as nested params can get kind of tricky sometimes. :+1:

**Comment by Ly900** [comment]
Yes, I definitely got to it and am able to work with the nested params now!

**Pull request closed without merging**